hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -968,6 +987,8 @@ private CopyObjectResponse copyObject(OzoneVolume volume,
       CopyObjectResponse copyObjectResponse = new CopyObjectResponse();
       copyObjectResponse.setETag(OzoneUtils.getRequestID());
       copyObjectResponse.setLastModified(destKeyDetails.getModificationTime());
+      getLatencyMetrics().addCopyObjectLatencyNs(
+          Time.monotonicNowNanos() - start);
```

Comment:
It's fine to do this way. However, it wouldn't record the latency if there's an exception and doesn't reach here.
Better to do it in the finally clause.

Alternatively, create a wrapper method around the code in copyObject(), and then you can add captureLatencyNs(() easily.

Reply:
Thank you for the suggestion, @jojochuang. I will be working on implementing the necessary modifications.